magazine-example/functions.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+if ( ! function_exists( 'magazine_example_support' ) ) :
+    function magazine_example_support()  {
+
+    	// Add featured image support.
+    	add_theme_support( 'post-thumbnails' );
+
+		// Make theme available for translation.
+		load_theme_textdomain( 'magazine-example', get_template_directory() . '/languages' );
+
+		// Let WordPress manage the document title.
+		add_theme_support( 'title-tag' );
+
+		// Alignwide and alignfull classes in the block editor
+		add_theme_support( 'align-wide' );
+
+		// Adding support for core block visual styles.
+		add_theme_support( 'wp-block-styles' );
+
+		// Adding support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+
+		// Add support for custom line height controls.
+		add_theme_support( 'custom-line-height' );
+
+		// Add support for experimental link color control.
+		add_theme_support( 'experimental-link-color' );
+
+		// Add support for editor styles.
+		add_theme_support( 'editor-styles' );
+
+		// Enqueue editor styles.
+		add_editor_style( 'editor-editor.css' );
```

Comment:
```suggestion
		add_editor_style( 'style-editor.css' );
```

Reply:
Perfect example of why it's good to have a second set of eyes on your code. 😄 Thank you!